src/components/Navigation/Sections/FilterBar.js

```diff
@@ -46,47 +47,49 @@ export default class FilterBar extends Component {
     } = this.props
 
     return (
-      <div className="list-filter" align="right">
-        {showSortFilter && (
-          <SortList
-            list={customSorts || sorts}
-            title={'Sort By'}
-            id={'sort'}
-            disabled={hasComponents}
-            value={activeSort}
-            onSort={onSort}
-          />
-        )}
-        {showLicenseFilter && (
-          <FilterList
-            list={customLicenses || licenses}
-            title={'License'}
-            id={'licensed.declared'}
-            disabled={hasComponents}
-            value={activeFilters}
-            onFilter={onFilter}
-          />
-        )}
-        {showSourceFilter && (
-          <FilterList
-            list={customSources || sources}
-            title={'Source'}
-            id={'described.sourceLocation'}
-            disabled={hasComponents}
-            value={activeFilters}
-            onFilter={onFilter}
-          />
-        )}
-        {showReleaseDateFilter && (
-          <FilterList
-            list={customReleaseDates || releaseDates}
-            title={'Release Date'}
-            id={'described.releaseDate'}
-            disabled={hasComponents}
-            value={activeFilters}
-            onFilter={onFilter}
-          />
-        )}
+      <div align="right">
+        <ButtonGroup>
```

Comment:
Adding a sub-level element in the DOM could affect the e2e tests, please check that everything is still working fine.

From the code i'm not able to understand what brings in the usage of a ButtonGroup here.
Does it allow to compact all the filters in a mobile view?